src/PooledArrays.jl

```diff
@@ -60,11 +70,14 @@ const PooledMatrix{T,R} = PooledArray{T,R,2}
 ##############################################################################
 
 # Echo inner constructor as an outer constructor
-function PooledArray(refs::RefArray{R}, invpool::Dict{T,R}, pool=_invert(invpool)) where {T,R}
```

Comment:
This constructor duplicated `R` and thus was never triggered as `RefArray` holds `RA` not `R`.